e2e/go.mod

```diff
@@ -14,4 +14,70 @@ require (
 	k8s.io/client-go v0.22.1
 )
 
+require (
```

Comment:
In golang v 1.17 the module graph has been changed to enable pruning and lazy loading. The second require block contains indirect dependencies. Ref: https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17, https://go.dev/doc/go1.17#go-command